site/js/modules/webui/course.js

```diff
@@ -35,12 +37,123 @@ function handlerCourse(path, params, context, container) {
         cards.push(Render.makeCardObject('assignment', assignment.name, link));
     }
 
+    cards.push(Render.makeCardObject(
+        "course-action",
+        "Email Users",
+        Routing.formHashPath(Routing.PATH_EMAIL, {
+            [Routing.PARAM_COURSE]: course.id,
+        }),
+    ));
+
     container.innerHTML = `
         <h2>${course.name}</h2>
         ${Render.cards(cards)}
     `;
 }
 
+function handlerEmail(path, params, context, container) {
+    let course = context.courses[params[Routing.PARAM_COURSE]];
+    const courseLink = Routing.formHashPath(Routing.PATH_COURSE, {[Routing.PARAM_COURSE]: course.id});
+
+    const titleHTML = `
+        <span>
+            <a href='${courseLink}'>${course.id}</a>
+            / email
+        </span>
+    `;
+    Routing.setTitle(course.id, titleHTML);
+
+    container.innerHTML = `
+        <div class="email-page">
+            <div class="email-content">
+                <h2>Email Users</h2>
+                <div class="description">
+                    <p>Sumbitting this form will send an email to all users enrolled in the course.</p>
```

Comment:
We don't want to use terminology like "submitting a form" for external text.
This is internal jargon that we can (and should) use when talking,
but not something that we should use when talking with a user
(normal people won't know what we mean).

Note that for admin-type things we can expect a little more from our uses,
but that is more for server admins, not course admins.